windows/CodePushLifecycleEventListener.cs

```diff
@@ -0,0 +1,44 @@
+﻿using ReactNative.Bridge;
+using System;
+
+namespace CodePush.ReactNative
+{
+    internal class CodePushLifecycleEventListener : ILifecycleEventListener
+    {
+        private DateTime? _lastSuspendDate = null;
+        private Action _loadBundleAction;
+
```

Comment:
If you did the above, then you could call this class something like "ThrottledResumeListener" or something more specific, without needing to be tied to CodePush per se. Not at all necessary, but it could help improve the semantics of the component.